easybuild/easyconfigs/l/LLVM/LLVM-12.0.0-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,43 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+name = 'LLVM'
+version = '12.0.0'
+
+homepage = "https://llvm.org/"
+description = """The LLVM Core libraries provide a modern source- and target-independent
+ optimizer, along with code generation support for many popular CPUs
+ (as well as some less common ones!) These libraries are built around a well
+ specified code representation known as the LLVM intermediate representation
+ ("LLVM IR"). The LLVM Core libraries are well documented, and it is
+ particularly easy to invent your own language (or port an existing compiler)
+ to use LLVM as an optimizer and code generator."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+toolchainopts = {'cstd': 'gnu++11'}
+
+source_urls = ['https://github.com/llvm/llvm-project/releases/download/llvmorg-%(version)s/']
+sources = ['llvm-%(version)s.src.tar.xz']
+checksums = ['49dc47c8697a1a0abd4ee51629a696d7bfe803662f2a7252a3b16fc75f3a8b50']
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+    ('CMake', '3.20.1'),
+    ('Python', '3.9.5'),
```

Comment:
@robert-mijakovic you can now use the `-bare` Python from https://github.com/easybuilders/easybuild-easyconfigs/pull/13035

Reply:
+1, makes more sense

Reply:
BTW, installing it with bare lead to sanity check failure.
```
== FAILED: Installation ended unsuccessfully (build directory: /dev/shm/build/Python/3.9.5/GCCcore-10.3.0-bare): build failed (first 300 chars): Sanity check failed: sanity check command python -c 'import _ssl' exited with code 1 (output: Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named '_ssl'
) (took 8 min 56 sec)
```

I guess I should open a ticket on that.

Reply:
I have opened a ticket on that https://github.com/easybuilders/easybuild-easyblocks/issues/2466